Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -950,6 +951,11 @@ bool RenderLayerCompositor::updateCompositingLayers(CompositingUpdateType update
         LOG(Compositing, "%8d %11d %9d %20.2f %22.2f %22.2f %18.2f\n",
             m_obligateCompositedLayerCount + m_secondaryCompositedLayerCount, m_obligateCompositedLayerCount,
             m_secondaryCompositedLayerCount, m_obligatoryBackingStoreBytes / 1024, m_secondaryBackingStoreBytes / 1024, (m_obligatoryBackingStoreBytes + m_secondaryBackingStoreBytes) / 1024, (endTime - startTime).milliseconds());
+
+#if ENABLE(RESOURCE_USAGE)
+        // set the total layer memory into ResourceUsageThread for Inspector layers memorytimeline.
```

Comment:
/* Unofficial Review */
// We save the total memory size of layers to show in the memory timeline of Inspector.
?